portable_fs/template/fs.py

```diff
@@ -11,6 +11,7 @@
 from . import core
 from collections import defaultdict
 import warnings
+import pathlib
```

Comment:
I assume that if you are not running [CI against py2k it is not supported](https://github.com/NSLS-II/portable-fs/blob/master/.travis.yml#L8)